getting-started/events.md

```diff
@@ -0,0 +1,268 @@
+  "author_id": "b7eff798-f8df-4364-8059-649c35c9ed0c",
+  "timestamp": 1473433500, // generated by server side
+  "text": "Example annotation",
+  "annotation_type": "rating",
+  "properties": {
+    // "Properties" object
+  }
+}
+```
+
+`annotation_type` - type of the annotation. This field cannot be empty and is customizable, you can have your own annotation types (i.e. we use "rating" type)
+
+
+### Filled form
+
+Filled form is event containing some data from form. Here's what it looks like:
+
+```js
+{
+  "id": "0affb00a-82d6-4e07-ae61-56ba5c36f743", // generated by server side
+  "custom_id": "12312.301231238591134",
+  "order": 4, // generated by server side
+  "type": "filled_form",
+  "author_id": "b7eff798-f8df-4364-8059-649c35c9ed0c",
+  "timestamp": 1473433500,  // generated by server side
+  "properties": {
+    // "Properties" object
+  }
+  "fields": [
+    // "Field" objects
+  ]
+}
+```
+
+There are 5 different types of field from filled form, all of these types can be mixed together in one filled form event.
+
+`text`:
+
+```js
+  {
+    "type": "text",
+    "name": "name",
+    "label": "Your name",
+    "required": true,
+    "value": "Jan Kowalski"
+  }
+```
+
+`email`:
+
+```js
+  {
+    "type": "email",
+    "name": "email",
+    "label": "Your email",
+    "required": true,
+    "value": "jan.kowalski@gmail.com"
+  }
+```
+
+`radio`:
+
+```js
+  {
+    "type": "radio",
+    "name": "purpose",
+    "label": "Chat purpose",
+    "required": true,
+    "options": [{
+      "label": "Support",
+      "value": "support",
+      "checked": true
+    },
+    {
+      "label": "Sale",
+      "value": "sale",
+      "checked": false
+    }],
+  }
+```
+
+`checkbox`:
+
+```js
+  {
+    "type": "checkbox",
+    "name": "industry",
+    "label": "Company industry",
+    "required": true,
+    "options": [{
+      "label": "automotive",
+      "value": "automotive",
+      "checked": true
+    }, {
+      "label": "IT",
+      "value": "it",
+      "checked": true
+    }],
+  }
+```
+
+`select`:
+
+```js
+{
+  "type": "select",
+  "name": "country",
+  "label": "Country",
+  "required": true,
+  "options": [{
+    "label": "USA",
+    "value": "usa",
+    "checked": false
+  }, {
+    "label": "Poland",
+    "value": "pl",
+    "checked": true
+  }, {
+    "label": "Poland",
+    "value": "pl",
+    "checked": false
+  }]
+}
+```
+
+
+### System message
+
+System message is event generated by server. It does not have "author_id", "custom_id" and "properties" fields.
+
+```js
+{
+  "id": "0affb00a-82d6-4e07-ae61-56ba5c36f743",
+  "order": 1,
+  "type": "system_message",
+  "timestamp": 1473433500,
+  "text": "Mike joined the chat",
+  "system_message_type": "agent_joined"
+}
+```
+
+System messages can be triggered by an user action within chat or by router (more about router messages in future Router section). Each action triggers system message with different `system_message_type` and has different `text` message:
```

Comment:
System messages can be triggered by an user action within chat or by router (more about router messages in future Router section). Each action triggers system message with different `system_message_type` and has different `text` message: -> System messages can be triggered by a user action within a chat or by a router (router messages will be covered in the future Router section). Each action triggers a system message with a different `system_message_type` and a different `text` message: